core/modules/proto/worker.proto

```diff
@@ -52,7 +52,14 @@ message TaskMsg {
         // contributing fragments' rows.
     }
     repeated Fragment fragment = 5;
-
+    message ScanTable {
+        required string db = 1;
+        required string table = 2;
+        required bool lockInMemory = 3;
+        required int32 scanSpeed = 4;
+    }
+    repeated ScanTable scantable = 4;
```

Comment:
Oh, you re-used 4 here.  That's considered bad form in protobufs (by not re-using, you ensure that mixes of older and newer code on the same network won't explode in flames).  Granted, here we are unlikely to mix czars and workers across versions.  Hmm...